Casks/t/ttu-base-suite.rb

```diff
@@ -0,0 +1,36 @@
+cask "ttu-base-suite" do
+  version "20.00.05.00,2024-01"
+  sha256 "b27c668a145edd80bf1d3adc93e0178e83ccc069a8aa0daaea324bcd77e369c1"
+
+  url "https://downloads.teradata.com/sites/default/files/#{version.csv.second}/TeradataToolsAndUtilities-macosx-brew-#{version.csv.first}.tar"
+  name "Teradata Tools and Utilities"
+  desc "Collection of Teradata client tools"
+  homepage "https://downloads.teradata.com/TeradataToolsAndUtilities_macOSBrew"
+
+  livecheck do
+    url "https://downloads.teradata.com/TeradataToolsAndUtilities_macOSBrew"
+    regex(%r{(\d+(?:-\d+)+)/TeradataToolsAndUtilities-macosx-brew-\s*(\d+(?:\.\d+)+)}i)
+    strategy :page_match do |page, regex|
+      page.scan(regex).map do |match|
+        "#{match[1]},#{match[0]}"
+      end
```

Comment:
```suggestion
    url :homepage
    regex(%r{href=.*?/(\d+(?:-\d+)+)/TeradataToolsAndUtilities-macosx-brew[._-]v?(\d+(?:\.\d+)+)\.t}i)
    strategy :page_match do |page, regex|
      page.scan(regex).map { |match| "#{match[1]},#{match[0]}" }
```

I'm good with this approach after some small tweaks.